grid/datasourcerequest-on-server/WasmApp/Server/Controllers/WeatherForecastController.cs

```diff
@@ -52,9 +52,10 @@ public async Task<DataEnvelope<WeatherForecast>> Post([FromBody]string dsRequest
             // in a real case, you would be fetching the data from the service, not generating it here
             IQueryable<WeatherForecast> queriableData = _forecasts.AsQueryable();
 
-            // deserialize the DataSourceRequest from the HTTP query
-			// make sure to use the System.Text.Json serializer
-            DataSourceRequest gridRequest = JsonSerializer.Deserialize<DataSourceRequest>(dsRequest);
+            // deserialize the DataSourceRequest from the HTTP query if it comes as a string like
+            // Post([FromBody]string dsRequest)
+            // make sure to use the System.Text.Json serializer
+            //DataSourceRequest gridRequest = JsonSerializer.Deserialize<DataSourceRequest>(dsRequest);
```

Comment:
This section also could be removed. I find that it was previously useful because of the string dsRequest parameter, however, the approach now is clear and intuitive.